shiftboard/settings.py

```diff
@@ -37,6 +37,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'board.apps.BoardConfig',
```

Comment:
I need to be emotionally moved by this code before I can let a pull request as controversial as this slip through